examples/pt2/torch_compile_openvino/README.md

```diff
@@ -0,0 +1,146 @@
+
+# TorchServe inference with torch.compile with OpenVINO backend of vgg16 model
+
+This example shows how to take eager model of `vgg16`, configure TorchServe to use `torch.compile` and run inference using `torch.compile` with [OpenVINO backend](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html) and compare the performance
+
+This example has been tested on Intel Xeon Platinum 8469 CPU.
+
+### Pre-requisites
+
+- `PyTorch >= 2.0`
```

Comment:
Do all the options work with 2.0 or does it need 2.2/2.3?

Reply:
All the options work with 2.0 or higher